lib/custom_types/openstack_query/aliases.py

```diff
@@ -1,23 +1,24 @@
-from typing import List, Callable, Any, Dict
+from typing import List, Callable, Any, Dict, Union, Literal
 from enum import Enum
 from enums.query.query_presets import QueryPresets
 
+OpenstackResourceObj = Any
 
-PropFunc = Callable[[Any], Any]
-PropToPropFuncMap = Dict[Enum, PropFunc]
+PropFunc = Callable[[OpenstackResourceObj], Dict[str, Any]]
```

Comment:
Nit: Is the `Dict[str, Any]` here related to any of the others?

I'm looking at `PresetKwargs` for example or `ServerSideFilters` to see if there's any overlap

Reply:
I renamed type aliases and added comments to them so they are less confusing

Reply:
It's much better now :+1: